clang/include/clang/Basic/LangOptions.def

```diff
@@ -398,11 +398,11 @@ BENIGN_LANGOPT(HalfNoSemanticInterposition, 1, 0,
                "Like -fno-semantic-interposition but don't use local aliases")
 ENUM_LANGOPT(StackProtector, StackProtectorMode, 2, SSPOff,
              "stack protector mode")
-ENUM_LANGOPT(TrivialAutoVarInit, TrivialAutoVarInitKind, 2, TrivialAutoVarInitKind::Uninitialized,
+BENIGN_ENUM_LANGOPT(TrivialAutoVarInit, TrivialAutoVarInitKind, 2, TrivialAutoVarInitKind::Uninitialized,
              "trivial automatic variable initialization")
-VALUE_LANGOPT(TrivialAutoVarInitStopAfter, 32, 0,
+BENIGN_VALUE_LANGOPT(TrivialAutoVarInitStopAfter, 32, 0,
              "stop trivial automatic variable initialization after the specified number of instances. Must be greater than 0.")
-VALUE_LANGOPT(TrivialAutoVarInitMaxSize, 32, 0,
+BENIGN_VALUE_LANGOPT(TrivialAutoVarInitMaxSize, 32, 0,
```

Comment:
I feel this may doesn't matter for header modules, but for named modules, it may affect generated codes. But I feel we should have other mechanism to detect and diagnose the inconsistent configs. (at least warning. Ideally emit diagnostics on need). I do meet such error diagnostics which stops people to use modules. So LGTM.

Reply:
The question here even for named modules is if it impacts the pcm file, not the .o. We already have the case where the pcm for a named module is built by an entirely different build system and compiler than the one that built the .o file associated with the module.